app/models/spree/gift_card.rb

```diff
@@ -30,7 +30,7 @@ def apply(order)
             adjustable: order,
             source: self,
             mandatory: true,
-            label: "#{Spree.t(:gift_card)}"
+            label: "#{Spree.t(:gift_card)} Discount"
```

Comment:
@AnuragBhardwaj1 Please move discount to I18n as well. Use Spree.t(:gift_card_discount) in subject